### PR TITLE
Add ability to hide cta on desktop to navbar

### PR DESCRIPTION
**Pull request by kenning**
necessary for https://app.asana.com/0/1116481661798430/1136005494202191

### CODE REVIEW

File: src/components/UniversalNavbar/UniversalNavbar.js [hunk 1]
```diff
@@ -194,15 +194,18 @@ class UniversalNavbar extends React.Component {
 
 UniversalNavbar.propTypes = {
   /** Hide cta on mobile viewport */
-  hideMobileCta: PropTypes.bool.isRequired,
+  hideMobileCta: PropTypes.bool,
+  /** Hide cta on desktop */
+  hideDesktopCta: PropTypes.bool,
   /** Run analytics function when CTA Button gets clicked */
-  trackCtaClick: PropTypes.func.isRequired,
+  trackCtaClick: PropTypes.func,
```
Comment (iamvictorli) [inline src/components/UniversalNavbar/UniversalNavbar.js]: I think this should be required, as long as defaultProps is defined.
From the site [React doc site](https://reactjs.org/docs/typechecking-with-proptypes.html):
> The propTypes typechecking happens after defaultProps are resolved, so typechecking will also apply to the defaultProps.

Comment (kenning) [reply src/components/UniversalNavbar/UniversalNavbar.js]: what's the point of marking them required if they have defaults?

Comment (kenning) [reply src/components/UniversalNavbar/UniversalNavbar.js]: i'll just do it but i am actually curious what you think about this

Comment (iamvictorli) [reply src/components/UniversalNavbar/UniversalNavbar.js]: You have a point. Never thought about that. There was this ESLint rule I used before, where everything had to be required or be in defaultProps. You have a point.

File: src/components/UniversalNavbar/UniversalNavbar.js [hunk 2]
```diff
@@ -203,6 +205,7 @@ UniversalNavbar.propTypes = {
 
 UniversalNavbar.defaultProps = {
   hideMobileCta: false,
+  hideDesktopCta: false,
```
Comment (roblevintennis) [inline src/components/UniversalNavbar/UniversalNavbar.js]: I agree with you Nick, let's just remove this if it's required anyway

File: src/components/UniversalNavbar/UniversalNavbar.js (hunk 1 above)
Comment (roblevintennis) [reply src/components/UniversalNavbar/UniversalNavbar.js]: > what's the point of marking them required if they have defaults?


Inversely, there's no reason to add a default if  something is required. For example:
[this eslint](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/require-default-props.md) states:

> Enforce a defaultProps definition for every prop **that is not a required prop** (react/require-default-props)

So I'd like us to remove the default values for required it's just adding more noise.

**Pull request merged**